lib/rules/lint-attribute-indentation.js

```diff
@@ -360,6 +414,9 @@ module.exports = class AttributeSpacing extends Rule {
       if ('indentation' in config) {
         result.indentation = config.indentation;
       }
+      if ('parse-html-elements' in config) {
```

Comment:
`html-elements` seems somewhat redundant (and we do theoretically support `svg` elements too). Maybe `process-elements`?

Reply:
Done.